src/lib/PriorityOrderLib.sol

```diff
@@ -24,18 +29,33 @@ struct PriorityOutput {
 struct PriorityOrder {
     // generic order information
     OrderInfo info;
+    // The address which must cosign the full order
+    address cosigner;
     // the block at which the order becomes active
-    uint256 startBlock;
+    uint256 auctionStartBlock;
+    // the block at which the order can be executed without a cosigner
+    uint256 openAuctionStartBlock;
+    // the minimum priority fee required for filling the order
```

Comment:
unless I'm misunderstanding, this is more like the "baseline" priority fee above which fees are taxed, no?

Reply:
I like baseline